user/languages/php.md

```diff
@@ -4,80 +4,111 @@ layout: en
+`X.Y` versions are aliases for recent `X.Y.Z` releases pre-installed on the build images.
+For exact versions used in your build, consult "Build system information" in the build log.
 
-- 5.4.x
-- 5.5.x
-- 5.6.x
-- 7.0.x
-- 7.1.x
-- hhvm # on Trusty only
-- nightly
+These may not be the most recent releases. If you need to ensure the use of most recent releases, do provide the third number; e.g.,
 
-which are specified using aliases for the "most recent x.y.z release" provided on Travis CI of any given line. For a full listing of the supported versions see [About Travis CI Environment](/user/reference/precise/).
+```yaml
+language: php
+php:
+  - 7.1.9
+```
+{: data-file=".travis.yml"}
 
-You can see an [example of version number aliases](https://github.com/travis-ci/travis-ci-php-example/blob/master/.travis.yml) on github. For precise versions used in your build, consult "Build system information" in the build log.
+### PHP 5.2(.x) and 5.3(.x) support is available on Precise only
+
+We do not suppport these versions on Trusty.
+If you need to test them, please use Precise.
+See [this page](/user/reference/trusty#PHP-images) for more information.
 
-> Specifying exact versions like 5.3.8 is discouraged as it may break your build when we update PHP versions on Travis CI. PHP version *5.5.9* is supported, however, because it's the version of PHP that is shipped with Ubuntu 14.04 LTS.
 
 ### HHVM versions
 
-Travis CI can test your PHP applications with HHVM on Ubuntu Trusty.
+Travis CI can test your PHP applications with HHVM on Ubuntu Trusty:
 
-Without specifying further, the latest version of HHVM available for the Ubuntu Trusty.
+```yaml
+php
+  - hhvm-3.18
+  - hhvm-nightly
+```
+
+### Nightly builds
 
-#### HHVM versions on Trusty
+Travis CI can test your PHP applications with a nightly
+[PHP](https://github.com/php/php-src/) build, which includes PHPUnit and
+Composer:
 
 ```yaml
 language: php
-sudo: required
-dist: trusty
-group: edge
+
 php:
-  - hhvm-3.3
-  - hhvm-3.6
-  - hhvm-3.9
-  - hhvm-3.12
-  - hhvm-3.15
-  - hhvm-3.18
-  - hhvm-nightly
+  - nightly
 ```
 {: data-file=".travis.yml"}
 
-## Default Test Script (PHPUnit)
+## Default Build Script
 
-The default test script is PHPUnit. It comes packaged with PHP, but you can also
+The default build script is PHPUnit. It comes packaged with PHP, but you can also
 install a specific version in a custom location. If you do install it
 separately, make sure you invoke the correct version by using the full path.
 
-Travis CI looks for `phpunit` in the [same order as Composer does](https://getcomposer.org/doc/articles/vendor-binaries.md#can-vendor-binaries-be-installed-somewhere-other-than-vendor-bin-)
+Travis CI looks for `phpunit` in the [same order as Composer
+does](https://getcomposer.org/doc/articles/vendor-binaries.md#can-vendor-binaries-be-installed-somewhere-other-than-vendor-bin-)
 and uses the first one found.
 
 1. `$COMPOSER_BIN_DIR/phpunit`
```

Comment:
after this, when we say
> If your project uses something other than PHPUnit, you can override the default test command.

how do you feel about updating it to: 
> If your project uses something other than PHPUnit, you can override the default build script command.

So that it's consistent with the whole section **Default Build Script**?

Reply:
@MariadeAnton Sounds good to me.